.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
```

Comment:
Is it a vscode config file? Is it a default name for it (sorry, never used it before)?

Reply:
Main question here: is this file auto-generated by ide? If yes, then it might become a source of unnecessary merge conflicts and should be in .gitignore.

Reply:
Has nothing to do with vscode, its not autogenerated.
You can read more about editorconfig tool at https://editorconfig.org/

Reply:
TIL, thanks.